test/pdfs/jbig2_huffman_1.pdf.link

```diff
@@ -0,0 +1 @@
+https://bugs.freedesktop.org/attachment.cgi?id=11143
```

Comment:
Please link to the Internet Archive instead, to avoid the file changing or becoming unavailable: https://web.archive.org/web/20171003213334/https://bugs.freedesktop.org/attachment.cgi?id=11143